common/persistence/visibility_triple_manager.go

```diff
@@ -232,13 +232,13 @@ func (v *visibilityTripleManager) UpsertWorkflowExecution(
 func (v *visibilityTripleManager) chooseVisibilityModeForAdmin() string {
 	switch {
 	case v.dbVisibilityManager != nil && v.sourceVisibilityManager != nil && v.destinationVisibilityManager != nil:
-		return common.AdvancedVisibilityWritingModeTriple
-	case v.dbVisibilityManager != nil && v.destinationVisibilityManager != nil:
-		return common.AdvancedVisibilityWritingModeDual
+		return common.AdvancedVisibilityMigrationWritingModeTriple
+	case v.sourceVisibilityManager != nil && v.destinationVisibilityManager != nil:
+		return common.AdvancedVisibilityMigrationWritingModeDual
 	case v.destinationVisibilityManager != nil:
-		return common.AdvancedVisibilityWritingModeOn
+		return common.AdvancedVisibilityMigrationWritingModeDestination
 	case v.dbVisibilityManager != nil:
-		return common.AdvancedVisibilityWritingModeOff
+		return common.AdvancedVisibilityMigrationWritingModeOff
```

Comment:
Is there a comment somewhere about the high-level intent of each of these options? Write to both?

Reply:
Yeah, this method is called in the chooseVisibilityManagerForWrite(), it has the switch case and determines which visibility datastore writes to.